examples/pulse_control/2_Native_gate_calibrations.ipynb

```diff
@@ -0,0 +1,418 @@
+{
```

Comment:
<p>How do we know for which parameters I can query a particular gate for?</p><p> </p><p>Let's split out the <code>print(pulse_sequence_xy_theta_q10_q113.to_ir())</code> to a separate cell and say that you can inspect the openpulse program corresponding to this PulseSequence by calling the <code>to_ir()</code> method on the PulseSequence.  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-braket-examples/pull/294/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>